data_management/models.py

```diff
@@ -667,16 +667,19 @@ class QualityControlled(BaseModel):
     ### Writable Fields:
     `object`: API URL of the associated `Object`
 
+    `field`: API URL of the `Object` representing the quality control document
```

Comment:
Could we change this to `document` instead of `field`?
```suggestion
    `document`: API URL of the `Object` representing the quality control document
```